src/py/flwr/client/app.py

```diff
@@ -141,6 +147,21 @@ def _parse_args_run_client_app() -> argparse.ArgumentParser:
         default="0.0.0.0:9092",
         help="Server address",
     )
+    parser.add_argument(
+        "--retry_max_tries",
```

Comment:
Is it possible to rename the flag to `--max-retries`, `--retries` or `--retry` I have a feeling that this might lead to confusion?